src/e2e/kotlin/fi/aalto/cs/apluscourses/e2e/fixtures/CommonFixtures.kt

```diff
@@ -95,6 +96,10 @@ class IdeFrameFixture(remoteRobot: RemoteRobot, remoteComponent: RemoteComponent
         byXpath("//div[@class='MyList']"),
         Duration.ofSeconds(20)
     )
+    fun codeWithMeButton() = findAll(
+        JButtonFixture::class.java,
+        byXpath("//div[@class='JButton' and @text='Got It']")
```

Comment:
Nit: Not very future-proof but okay. (OTOH, even if IntelliJ changed, it wouldn't probably do any harm to click buttons that say "Got It").